GSA/digital-gov-drupal
# DIGITAL-143: Update topics to autocomplete field

## Jira ticket




[DIGITAL-143](https://cm-jira.usa.gov/browse/DIGITAL-143)

## Purpose



This PR will update the topics field from inline form to autocomplete

## Includes the following PRs that must be merged first



## Deployment and testing

### Local Setup



`lando rebuild -y`

### QA/Testing instructions



1. Visit ` events, guides, news, resources, short post` and check that topics is now autocomplete.

## Checklist for the Developer




## Checklist for the Peer Reviewers
